packages/mui-material/src/PaginationItem/PaginationItem.test.js

```diff
@@ -14,7 +15,7 @@ describe('<PaginationItem />', () => {
 
   describeConformance(<PaginationItem />, () => ({
     classes,
-    inheritComponent: 'button',
+    inheritComponent: ButtonBase,
```

Comment:
PaginationItem is actually built on top of `ButtonBase` not `button`

https://github.com/mui/material-ui/blob/e8247891ff3ad9a60a4786dc74636a6c838cf7ab/packages/mui-material/src/PaginationItem/PaginationItem.js#L107